community/lms/doctype/lms_batch/lms_batch.json

```diff
@@ -116,6 +122,25 @@
    "fieldtype": "Time",
    "in_list_view": 1,
    "label": "End Time"
+  },
+  {
+   "fieldname": "telegram_link",
+   "fieldtype": "Data",
+   "label": "Telegram Link"
+  },
+  {
+   "fieldname": "video_call_link",
+   "fieldtype": "Data",
+   "label": "Video Call Link"
+  },
```

Comment:
Rather than hardcoding `telegram_link` and `video_call_link`, it may be a good idea to add generic links or add a field `description` of type markdown to allow the mentor to decide what ever they want to show there. It would be possible to  include links to video conferenrence, telegram or something else.